lxd/storage_volumes_config.go

```diff
@@ -72,10 +72,6 @@ func storageVolumeValidateConfig(name string, config map[string]string, parentPo
 			if config["block.filesystem"] == "" {
 				config["block.filesystem"] = parentPool.Config["volume.block.filesystem"]
 			}
```

Comment:
Similarly to what I did for the pools, this chunk should be moved to storageVolumeFillDefault and storageVolumeFillDefault shouldn't be called by storageVolumeValidateConfig but instead be called only when actually needed.